plugins/apple/swift/v1.29.0/buf.plugin.yaml

```
@@ -0,0 +1,21 @@
+version: v1
+name: buf.build/apple/swift
+plugin_version: v1.29.0
+source_url: https://github.com/apple/swift-protobuf
+integration_guide_url: https://github.com/apple/swift-protobuf#getting-started
+description: Base types for Swift. Generates message and enum types.
+output_languages:
+  - swift
+registry:
+  swift:
+    deps:
+      - source: https://github.com/apple/swift-protobuf.git
+        package: swift-protobuf
+        swift_versions: [ ".v5" ]
```

It's a tad tricky to tell, but I think we can add `".v6"` here since https://github.com/apple/swift-protobuf/pull/1740 is included in [this release](https://github.com/apple/swift-protobuf/releases/tag/1.29.0)?

```suggestion
        swift_versions: [ ".v5", ".v6" ]
```